tests/mixins/responsiveManagerMixin-test.js

```diff
@@ -0,0 +1,63 @@
+jest.dontMock('../../js/mixins/responsiveManagerMixin');
+jest.dontMock('../../js/components/utils');
+
+const React = require('react');
+const Enzyme = require('enzyme');
+const debounce = require('lodash.debounce');
+const createReactClass = require('create-react-class');
+const responsiveManagerMixin = require('../../js/mixins/responsiveManagerMixin');
+
+let Foo = createReactClass({
+  mixins: [responsiveManagerMixin],
+  render: function() {
+    return (
+      <div>{this.props.children}</div>
+    );
+  }
+});
+
+describe('responsiveManagerMixin', function() {
+  let props =  {
```

Comment:
const

Reply:
No, it may be changeable in future, if someone wants to add tests for the file.